hw/ip/hmac/data/hmac.prj.hjson

```diff
@@ -5,8 +5,8 @@
 {
     name:               "hmac",
     version:            "0.5",
-    life_stage:         "L1",
+    life_stage:         "L2",
     design_stage:       "D3",
-    verification_stage: "V2",
+    verification_stage: "V3",
     notes:              "D3 commit id: 3471a47c96ba87c77de7b14c39f4faf71a839283",
```

Comment:
I will add another PR to update the V3 commit ID once this PR is merged.

Reply:
make sure you use the commit_id field.

Reply:
Will do, thank you for the reminder.